gptscript/opts.py

```diff
@@ -30,6 +30,7 @@ def __init__(self,
                  chatState: str = "",
                  confirm: bool = False,
                  prompt: bool = False,
+                 credentialOverride: str = "",
```

Comment:
This is outside the scope of this PR, but the SDK is using `camelCase` which, IIUC, is not pythonic. At some point we should probably look into refactoring to `snake_case`.

Reply:
I understand. I didn't want to do this because all of this get marshaled in the Go code. I would prefer to break the `snake_case` paradigm rather than write a bunch of custom serialization code.

Reply:
I suspected that was the case. Thanks.